astroquery/eso/core.py

```diff
@@ -207,6 +207,8 @@ def list_instruments(self, cache=True):
                     if instrument not in self._instrument_list:
                         self._instrument_list.append(instrument)
             self._instrument_list.append(u'harps')
+            self._instrument_list.append(u'ambient_paranal')
+            self._instrument_list.append(u'meteo_paranal')
```

Comment:
I don't see HARPS there.  As far as I can tell, it's not a browser cache issue.  So, you're right, it's VLT-only... maybe we should ask ESO to provide one for non-VLT instruments (since they provide the query forms...)